parm/mom6/MOM_input_template_050

```diff
@@ -98,7 +98,7 @@ WRITE_GEOM = 2                  ! default = 1
                                 ! If =0, never write the geometry and vertical grid files. If =1, write the
                                 ! geometry and vertical grid files only for a new simulation. If =2, always
                                 ! write the geometry and vertical grid files. Other values are invalid.
-SAVE_INITIAL_CONDS = True      !   [Boolean] default = False
+SAVE_INITIAL_CONDS = False      !   [Boolean] default = False
```

Comment:
See above.

Reply:
Yes, this file should match what is here: https://github.com/ufs-community/ufs-weather-model/blob/Release/P8b/tests/parm/MOM_input_template_050